projects/core/src/state/utils/scoped-loader/entity-scoped-loader.actions.ts

```diff
@@ -86,13 +89,32 @@ export namespace EntityScopedLoaderActions {
     implements EntityScopedLoaderAction, ErrorAction
   {
     type = ENTITY_FAIL_ACTION;
-    error: ErrorActionType;
+    error: ActionErrorProperty;
     readonly meta: EntityScopedLoaderMeta;
 
+    /**
+     * @deprecated Please use `error` parameter other than `null` or `undefined`.
+     *
+     *             Note: Allowing for `null` or `undefined` will be removed in future versions
+     *             together with the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.
+     **/
+    constructor(
```

Comment:
`query:` Shouldn't the eslint-disable-next-line comment be used in a deprecated constructor? Or actually it doesn't matter? Nevertheless, it's different than in other places.